src/pages/Home/sections/OpenSource/OpenSource.module.scss

```diff
@@ -0,0 +1,119 @@
+.OpenSource {
+  text-align: center;
+  margin-top: 7rem;
+
+  .title {
+    font-family: "Poppins", sans-serif;
+    color: #4e89ae;
+    font-size: 3rem;
+    font-weight: bold;
+    text-shadow: 3px 4px 2px #0000002e;
```

Comment:
colour code is not hexadecimal

Reply:
`#000000` represents black `2e` represents the opacity

Reply:
`#0000002e` is a valid color